service/src/main/java/uk/gov/hmcts/reform/fpl/service/cafcass/api/CafcassApiSearchCaseService.java

```diff
@@ -51,19 +54,25 @@ public List<CafcassApiCase> searchCaseByDateRange(LocalDateTime startDate, Local
 
         return caseDetails.stream()
             .map(this::convertToCafcassApiCase)
+            .filter(Objects::nonNull)
             .toList();
     }
 
     private CafcassApiCase convertToCafcassApiCase(CaseDetails caseDetails) {
-        return CafcassApiCase.builder()
-            .caseId(caseDetails.getId())
-            .jurisdiction(caseDetails.getJurisdiction())
-            .state(caseDetails.getState())
-            .caseTypeId(caseDetails.getCaseTypeId())
-            .createdDate(caseDetails.getCreatedDate())
-            .lastModified(caseDetails.getLastModified())
-            .caseData(getCafcassApiCaseData(caseConverter.convert(caseDetails)))
-            .build();
+        CaseData caseData = caseConverter.convert(caseDetails);
+        if (featureToggleService.isCafcassAPIEnabled(caseData.getCourt())) {
```

Comment:
Can we filter out directly in the `searchCaseQuery`? I think the `court` field has Searchable: Y so should be able to be queried 🤞